crates/core/src/otel.rs

```diff
@@ -32,6 +32,8 @@ pub struct OtelConfig {
     /// Determines whether http or grpc will be used for exporting the telemetry.
     #[serde(default)]
     pub protocol: OtelProtocol,
+    /// Additional CAs to include in the OpenTelemetry client configuration
+    pub additional_ca_paths: Vec<PathBuf>,
```

Comment:
Please mark this as `#[serde(default)]` for backwards compatibility in providers

Reply:
Good catch, thanks!